resolver/addressable_resolver.go

```diff
@@ -163,7 +163,7 @@ func (r *URIResolver) URIFromObjectReference(ref *corev1.ObjectReference, parent
 		return url, nil
 	}
 
-	_, lister, err := r.informerFactory.Get(gvr)
+	_, lister, err := r.informerFactory.Get(context.Background(), gvr)
```

Comment:
Do we need to plumb?

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616